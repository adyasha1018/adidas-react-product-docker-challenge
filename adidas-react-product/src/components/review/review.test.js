import React from "react";
import { shallow } from "enzyme";
import ReviewComponent from "./review.component";
describe("Review", () => {
  const props = {
      id:"H332",
      reviews:[
        {
          productId: "H332",
          locale: "en-US,en;q=0.9",
          rating: 4,
          text: "this is review",
        },
      ]
  };
  const wrapper = shallow(<ReviewComponent {...props}/>);
  it("should render ReviewComponent", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
