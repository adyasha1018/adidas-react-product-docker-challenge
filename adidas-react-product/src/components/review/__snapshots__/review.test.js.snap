// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Review should render ReviewComponent 1`] = `ShallowWrapper {}`;
